.github/workflows/scale-test-100-gce.yaml

```diff
@@ -52,22 +57,38 @@ jobs:
       - name: Set Environment Variables
         uses: ./.github/actions/set-env-variables
 
-      - name: Get Cilium's default values
-        id: default_vars
-        uses: ./.github/actions/helm-default
-        with:
-          image-tag: ${{ github.sha }}
```

Comment:
Would it be better to keep the default values action and just provide the override values we want to use? This is a pattern we follow in other workflows, and I think will make it clearer which values we want to differ from the defaults; ex: we want monitor aggregation turned on, but it's not mentioned in the workflow

Reply:
I would argue we are using "more" default values in this workflow (from users point of view) than other workflows that use `./.github/actions/helm-default`.
For example, just recently `./.github/actions/helm-default` was updated with `dnsProxy.proxyResponseMaxDelay` or `hubble.relay.retryTimeout`.
I personally think that `helm-default` is essentially the default config for testing the correctness of cilium, not necessarily performance, with verbose logging, monitoring events etc. I would prefer to have more consistent results so `./.github/actions/helm-default` changes do not affect it. WDYT?

Reply:
Makes sense to me :+1: